easybuild/easyconfigs/v/Vampir/Vampir-8.4.1-demo-linux-x86_64-foss-2015a.eb

```diff
@@ -0,0 +1,48 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2013-2015 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+#             Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock='Binary'
+
+name = "Vampir"
+version = "8.4.1"
+versionsuffix = "-demo"
+variant = "linux-x86_64"
```

Comment:
This doesn't make sense. If we ever want to include easyconfigs for another architecture but the same version of Vampir, we need two different easyconfigs with just a different variant and obviously a different file name. Why include this in the `versionsuffix`? IMHO there is no need to, as installations for different architectures are typically not shared.